src/main/java/org/shanerx/tradeshop/commands/CommandTabCompleter.java

```diff
@@ -71,7 +71,7 @@ public CommandTabCompleter(TradeShop instance, CommandPass command) {
 
 	public List<String> addSet() {
 		if (command.argsSize() == 2) {
-			return Arrays.asList("1", "2", "4", "8", "16", "32", "64", "96", "128");
+			return Arrays.asList("1", "2", "4", "8", "16", "32", "64", "80", "96", "128");
```

Comment:
These are tab completion values for amounts, 80 is a stack + 16. Tikkle requested it be added as it was am amount that his players used. These are strings because that's what the tab completion uses and putting them as ints to just be converted would be unnecessary and wasteful.

Reply:
Fair enough